src/Microsoft.SqlTools.SqlCore/Microsoft.SqlTools.SqlCore.csproj

```diff
@@ -12,6 +12,12 @@
 		<PreserveCompilationContext>true</PreserveCompilationContext>
 		<AssemblyTitle>SqlTools SqlCore Library</AssemblyTitle>
 		<Description>Provides core sql functionality for SQL server editors like Object explorer, Query Execution and Scripting</Description>
+		<TargetFramework>netstandard2.0</TargetFramework>
+		<LangVersion>8.0</LangVersion>
+		<AssemblyName>Microsoft.SqlTools.SqlCore</AssemblyName>
+		<Authors>Microsoft</Authors>
+		<Version>1.0.0</Version>
+		<Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
```

Comment:
We already define this in https://github.com/microsoft/sqltoolsservice/blob/main/Directory.Build.props#L8, so you shouldn't need to redefine these common props